src/Symfony/Bundle/WebProfilerBundle/EventListener/WebDebugToolbarListener.php

```diff
@@ -89,8 +92,6 @@ public function onKernelResponse(ResponseEvent $event)
             return;
         }
 
-        $nonces = $this->cspHandler ? $this->cspHandler->updateResponseHeaders($request, $response) : [];
```

Comment:
Will this still properly disable CSP for WebProfiler pages where we currently disable CSP entirely but we don't inject the toolbar ?

Reply:
Good catch, it did not. I reverted it, moved the check up and tested it manually and now it's fine.